models/intermediate/field_history/int_jira__daily_field_history.sql

```diff
@@ -26,9 +26,9 @@ limit_to_relevant_fields as (
     from combined_field_histories
 
     where 
-    lower(field_name) in ('sprint', 'status' 
+    lower(field_id) in ('sprint', 'status' 
```

Comment:
in our data sprint has a custom field ID (instead of `sprint` it's `customfield_10020`)

Reply:
Great catch @fivetran-jamie! I just pushed a new commit that accommodates for the sprint field being custom. Let me know if you have any questions. The solution is pretty basic, but works as intended!